lib/Raven/Breadcrumbs/ErrorHandler.php

```diff
@@ -41,7 +41,11 @@ public function handleError($code, $message, $file = '', $line = 0, $context = a
 
     public function install()
     {
-        $this->existingHandler = set_error_handler(array($this, 'handleError'), E_ALL);
+        $this->existingHandler = set_error_handler(
```

Comment:
As I said below, [callable](http://php.net/manual/en/language.types.callable.php) is not old. Lambda is not appropriate in that case because it introduces an overhead for no reason. One thing is to have to call a function (in that case `handleError`) which takes a different set of arguments than the input, so you can proxy the call through a lambda (this is just an example) and bind different arguments or adds more, one thing is call a lambda just to call another function that takes the same arguments and can be called with one line and the standard PHP syntax. I'm sorry, but what you're doing have no pros and a lot of cons